packages/protocol/contracts/signal/ISignalService.sol

```diff
@@ -8,25 +8,28 @@
 pragma solidity ^0.8.20;
 
 /**
+ * The SignalService contract serves as a secure cross-chain message passing
+ * system. It defines methods for sending and verifying signals with merkle
+ * proofs. The trust assumption is that the target chain has secure access
+ * to the merkle root (such as Taiko injects it in the anchor transaction).
+ * With this, verifying a signal is reduced to simply verifying a merkle proof.
```

Comment:
will put @title as the first lien better?

Reply:
overall, i agree with this. in fact, there are two changes i want to make:
- @title before description
- verbosely include "@notice" even though it is not required by solidity compiler
- move multi-line comment blocks to `///` instead of `/** */` for simplicity

however, i would like to introduce that later, after we cleanup the bridge documentation as well and get consistency throughout the codebase. i already commented the bridge/signal service previously using the standards here: https://taiko.xyz/docs/manuals/contributing-manual#source-code-comments

so don't want to bikeshed over this. once we document to the spec, we can update the spec and use chatGPT to fixup all the files quickly.